modules/platforms/cpp/tests/test-common/ignite_runner.cpp

```diff
@@ -30,8 +30,8 @@ namespace {
 /**
  * System shell command string.
  */
-constexpr std::string_view SYSTEM_SHELL = IGNITE_SWITCH_WIN_OTHER("cmd.exe", "/bin/sh");
-constexpr std::string_view SYSTEM_SHELL_ARG_0 = IGNITE_SWITCH_WIN_OTHER("/c ", "-c");
+constexpr std::string_view SYSTEM_SHELL = IGNITE_OS_SWITCH("cmd.exe", "/bin/sh", "/bin/sh");
+constexpr std::string_view SYSTEM_SHELL_ARG_0 = IGNITE_OS_SWITCH("/c ", "-c", "-c");
```

Comment:
Original macro `IGNITE_SWITCH_WIN_OTHER` would fit perfectly here, why change it just to duplicate the same code for Linux and macOS?

Reply:
Fixed